src/tests/Microsoft.PowerFx.Core.Tests/ExpressionTestCases/CustomObject.txt

```diff
@@ -1,17 +1,24 @@
-﻿>> Value(Index(ParseJson("[5]"), 1))
+﻿// Value tests
+>> Value(Index(ParseJson("[5]"), 1))
```

Comment:
>Value [](http://example.com/codeflow?start=3&length=5)

add test : 
equality ops on CO is an error. 
x = ParseJson("5"); x= x  is an error.

Reply:
Do we want to have negative tests everywhere in the language for things we do not support (and plan to never support)?

Reply:
Not "everywhere"  - that could be a never ending chore. 

But things that we explicitly had design discussions on and agreed should fail are good candidates for testing.  It gives an opportunity to capture the reasoning, ensure it doesn't accidentally work (perhaps some other bug, or coercion that wrongfully kicks in)  and prevent regressions. 
Especially protects us against a future dev that wants to "helpfully" enable it.